components/autocomplete/events.md

```diff
@@ -227,6 +228,62 @@ When using `OnRead`, make sure to set `TItem` and `TValue`.
 }
 ````
 
+## OnItemRender
+
+The `OnItemRender` event fires when each item in the collection passed to the `Data` parameter of the AutoComplete renders. 
+
+The event handler receives as an argument an `AutoCompleteItemRenderEventArgs<TItem>` object that contains:
+
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+| Property | Description                              |
+|----------|------------------------------------------|
+| `Item`   | The current item that renders in the AutoComplete. |
+| `Class`  | The custom CSS class that will be added to the item.     |
```

Comment:
We should either use ASCII art properly (which requires maintenance when the table contents changes), or remove it completely. This table was somewhere in the middle.

```suggestion
| Property | Description |
| --- | --- |
| `Item` | The current item that renders in the AutoComplete. |
| `Class` | The custom CSS class that will be added to the item. |
```